packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/logs-inputs.service.js

```diff
@@ -411,6 +414,12 @@ class LogsInputsService {
       [input.info.engine.name, input.info.engine.version].join(' '),
     );
     set(input, 'info.exposedPort', parseInt(input.info.exposedPort, 10));
+    if (
+      has(input, 'info.allowedNetworks.length') &&
```

Comment:
Isn't it the same ?

```js
has(input, 'info.allowedNetworks.length')  === Array.isArray(input.info.allowedNetworks)
```

Reply:
@FredericEspiau a string could have a length

Reply:
@marie-j because a string is an Array 😅 

What I meant is: isn't `Array.isArray` enough ?

Reply:
You are right, Array.isArray is enough.